app/src/main/java/org/p2p/wallet/home/repository/HomeDatabaseRepository.kt

```diff
@@ -27,10 +26,7 @@ class HomeDatabaseRepository(
 
     override fun getTokensFlow(): Flow<List<Token.Active>> =
         tokenDao.getTokensFlow()
-            .map { entities ->
-                entities.map { TokenConverter.fromDatabase(it) }
-                    .sortedWith(TokenComparator())
-            }
+            .map { it.map(TokenConverter::fromDatabase) }
```

Comment:
Where did you apply sorting?

Reply:
all the sorting happens in the interactor, as it was really before